xbmc/platform/darwin/ios-common/DarwinEmbedNowPlayingInfoManager.mm

```diff
@@ -29,8 +29,20 @@ - (void)onPlay:(NSDictionary*)item
   auto dict = [NSMutableDictionary new];
 
   NSString* title = item[@"title"];
+  NSString* epg_title = item[@"epg_title"];
   if (title && title.length > 0)
-    dict[MPMediaItemPropertyTitle] = title;
+  {
+    if (epg_title && epg_title.length > 0)
+    {
+      // Use artist property to put the channel name
+      // and title field for the TV show name.
+      dict[MPMediaItemPropertyTitle] = epg_title;
```

Comment:
I think the usage of epg_title can be removed as well. You fill both an Artist/Title based on epg_title.length > 0, so maybe do the check in the other class where you set the epg_title dict item. Just fill title and artist dict items with the data you want there, then this whole block goes away, as it just parses the existing title/artist dict items